crates/sui-types/src/messages_checkpoint.rs

```diff
@@ -207,9 +211,21 @@ impl CheckpointSummary {
         epoch_rolling_gas_cost_summary: GasCostSummary,
         end_of_epoch_data: Option<EndOfEpochData>,
         timestamp_ms: CheckpointTimestamp,
+        randomness_rounds: Vec<RandomnessRound>,
     ) -> CheckpointSummary {
         let content_digest = *transactions.digest();
 
+        let version_specific_data = match protocol_config
+            .checkpoint_summary_version_specific_data_as_option()
+        {
+            None | Some(0) => Vec::new(),
```

Comment:
So here (or somewhere), I think we also need to check that when `checkpoint_summary_version_specific_data` is < 0, `min_checkpoint_interval_ms` must be 0 . Otherwise, if checkpoint_summary_version_specific_data < 1, and min_checkpoint_interval_ms > 0, randomness rounds may be missed here.